### PR TITLE
Adding AvailableTabProducts policy

### PR DESCRIPTION
**Pull request by Tanuja-Kulal**
Adding AvailableTabProducts policy which is a string field. This will contain product Ids like 'Microsoft.VisualStudio.Product.Professional"

Testing- Tested by following readme 

image

image

### CODE REVIEW

File: settingFiles/admx/VisualStudio.admx [hunk 1]
```diff
@@ -377,6 +377,22 @@
...
+      <policy
+          name="AvailableTabProducts"
+          class="Machine"
+          displayName="$(string.AvailableTabProducts_DisplayName)"
+          explainText="$(string.AvailableTabProducts_Explain)"
+          presentation="$(presentation.AvailableTabProducts_PresentationId)"
+          key="SOFTWARE\Policies\Microsoft\VisualStudio\Setup"
+          valueName="AvailableTabProducts">
+        <parentCategory ref="InstallandUpdateSettings" />
+        <supportedOn ref="VisualStudio2022_12AndHigher" />
```
Comment (christineruana) [inline settingFiles/admx/VisualStudio.admx:389]: should this be VisualStudio2022_13AndHigher?

File: settingFiles/admx/en-US/VisualStudio.adml [hunk 2]
```diff
@@ -159,7 +159,11 @@ If set to a 1, the update notification in the Visual Studio IDE will be suppress
 
 For more information, see https://aka.ms/vs/setup/policies.</string>
 
-		<!-- LiveShare -->
+      <string id="AvailableTabProducts_DisplayName">Filter and restrict Available tab products in Visual Studio Installer</string>
```
Comment (christineruana) [inline settingFiles/admx/en-US/VisualStudio.adml:162]:
```suggestion
      <string id="AvailableTabProducts_DisplayName">Restrict products that show up in the Visual Studio installer's Available tab.</string>
```

File: settingFiles/admx/en-US/VisualStudio.adml [hunk 3]
```diff
@@ -159,7 +159,11 @@ If set to a 1, the update notification in the Visual Studio IDE will be suppress
...
+      <string id="AvailableTabProducts_Explain">Add product ids to restrict list shown in Available Tab of Visual Studio Installer. e.g. Microsoft.VisualStudio.Product.Professional. For more information, see  https://aka.ms/vs/setup/policies.</string>
```
Comment (christineruana) [inline settingFiles/admx/en-US/VisualStudio.adml:163]:
```suggestion
      <string id="AvailableTabProducts_Explain">This policy can be used to restrict which Visual Studio product shows up on the Installer's Available Tab. For example, including Microsoft.VisualStudio.Product.Professional in the list will cause the Visual Studio Professional edition to be the only available option to install. This policy is only applicable for client machines that have only the Visual Studio installer installed. For more information, see  https://aka.ms/vs/setup/policies.</string>
```

**Review by christineruana — DISMISSED** [review]

File: settingFiles/admx/VisualStudio.admx (hunk 1 above)
Comment (edwardskrod) [reply settingFiles/admx/VisualStudio.admx]: @Tanuja-Kulal  I took care of this.

**Review by edwardskrod — APPROVED** [review]
Looks good.  I committed a few of Christine's verbiage changes and fixed a few small grammatical errors.

**Comment by Tanuja-Kulal** [comment]
> Looks good. I committed a few of Christine's verbiage changes and fixed a few small grammatical errors.

Thanks

**Review by edwardskrod — DISMISSED** [review]

**Comment by jlee671** [comment]
I'm assuming the screenshot text is just outdated since we fixed the version from 12 to 13. The insertion PR to VS that contains your change just completed today, please test the functionality when it's available for testing.

**Review by jlee671 — APPROVED** [review]

**Pull request merged**